README.md

```diff
@@ -1,63 +1,109 @@
-# Cross-Language Test Suite for Dredd Hooks
+# Cross-Language Test Suite for Dredd Hooks Handlers
 
-[![Build Status](https://travis-ci.org/apiaryio/dredd-hooks-template.svg?branch=master)](https://travis-ci.org/apiaryio/dredd-hooks-template) [![Greenkeeper badge](https://badges.greenkeeper.io/apiaryio/dredd-hooks-template.svg)](https://greenkeeper.io/)
+[![Build Status](https://travis-ci.org/apiaryio/dredd-hooks-template.svg?branch=master)](https://travis-ci.org/apiaryio/dredd-hooks-template)
 
-Language-agnostic BDD test suite for boilerplating implementation of [Dredd][] [hooks][] handler for a new language. It tests the public interface of the hooks handler and ensures it will work as Dredd expects. It's written in [Gherkin][] and ran by [Aruba][].
-
-  [Aruba]: https://github.com/cucumber/aruba
-  [Gherkin]: https://github.com/cucumber/cucumber/wiki/Gherkin
-  [Dredd]: https://github.com/apiaryio/dredd
-  [hooks]: https://dredd.readthedocs.io/en/latest/hooks/
+[Dredd](https://dredd.org) is a tool for testing web APIs. It supports [hooks](http://dredd.org/en/latest/hooks/index.html) written in [many languages](http://dredd.org/en/latest/hooks/index.html#supported-languages). To support a particular language, it needs an adapter, so-called hooks handler. This [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development) test suite ensures that the public interface of each hooks handler works as Dredd expects. The test suite is written in [Gherkin](https://github.com/cucumber/cucumber/wiki/Gherkin) and uses [Cucumber](https://github.com/cucumber/cucumber-js) as a test runner.
 
 ## Usage
 
-### Install the features dependencies
+First read the docs about [how to create a new hooks handler](http://dredd.org/en/latest/hooks/new-language.html) for Dredd. Implement your hooks handler and then make sure it passes this test suite.
 
-```bash
-npm install -g dredd
-git clone https://github.com/apiaryio/dredd-hooks-template.git
-cd dredd-hooks-template
-bundle install
-```
+### Installation
+
+1.  Make sure you have [Node.js](https://nodejs.org/) and [npm](https://www.npmjs.com/package/npm) available.
```

Comment:
No specific versions of Node? Maybe we should mention the lowest supported version.